mage_ai/frontend/components/datasets/overview/index.tsx

```diff
@@ -159,25 +160,62 @@ function DatasetOverview({
     });
   };
 
-  return (
-    <Layout
-      centerAlign
-      footer={<Spacing mt={UNIT} />}
+  const closeAction = () => setActionPayload({} as ActionPayloadType);
+
+  const selectActionEl = (
+    <Select
+      // @ts-ignore
+      compact
+      onChange={e => setActionPayload(JSON.parse(e.target.value))}
+      value={actionType}
+      width={UNIT * 20}
     >
-      <Spacing mt={UNIT}>
-        {actionType && (
-          <ActionForm
-            actionType={actionType}
-            axis={actionPayload?.axis}
-            features={features}
-            onSave={() => saveAction({
-              action_payload: {
-                ...actionPayload,
-                action_type: actionType,
-              },
-            })}
-            payload={actionPayload}
-            setPayload={setActionPayload}
+      <option value="">
+        New action
+      </option>
+
+      {Object.entries(actionsConfig.rows).map(([k, v]) => (
+        <option
+          key={k}
+          value={JSON.stringify({
+            action_type: k,
+            axis: 'row',
```

Comment:
there's an `AxisEnum` we can use in the `mage-ai/mage_ai/frontend/interfaces/ActionPayloadType.ts` file

Reply:
instead of hardcoding 'row' here

Reply:
think it was added after we built out the action form